src/functionalTest/java/uk/gov/hmcts/reform/pip/data/management/location/LocationTest.java

```diff
@@ -0,0 +1,180 @@
+
+        String filePath = this.getClass().getClassLoader().getResource("location/ValidLocations.csv").getPath();
+        File csvFile = new File(filePath);
+
+        final Response responseUploadLocations = doPostRequestMultiPart(
+            UPLOAD_LOCATIONS_URL,
+            headerMapUploadLocations, LOCATION_LIST, csvFile
+        );
+
+        assertThat(responseUploadLocations.getStatusCode()).isEqualTo(OK.value());
+        assertThat(responseUploadLocations.as(Location[].class)).hasSize(3);
+
+        List createdLocations = Arrays.asList(responseUploadLocations.getBody().as(Location[].class));
+
+        assertThat(((Location) createdLocations.get(0)).getLocationId()).isEqualTo(TEST_LOCATION_ID_ONE);
+        assertThat(((Location) createdLocations.get(0)).getName()).isEqualTo(TEST_LOCATION_NAME_ONE);
+        assertThat(((Location) createdLocations.get(1)).getLocationId()).isEqualTo(TEST_LOCATION_ID_TWO);
+        assertThat(((Location) createdLocations.get(1)).getName()).isEqualTo(TEST_LOCATION_NAME_TWO);
+        assertThat(((Location) createdLocations.get(2)).getLocationId()).isEqualTo(TEST_LOCATION_ID_THREE);
+        assertThat(((Location) createdLocations.get(2)).getName()).isEqualTo(TEST_LOCATION_NAME_THREE);
+
+        final Response responseGetLocationById = doGetRequest(
+            GET_LOCATION_BY_ID_URL,
+            headerMap
+        );
+
+        assertThat(responseGetLocationById.getStatusCode()).isEqualTo(OK.value());
+        Location returnedLocation = responseGetLocationById.as(Location.class);
+        assertThat(returnedLocation.getLocationId()).isEqualTo(TEST_LOCATION_ID_ONE);
+        assertThat(returnedLocation.getName()).isEqualTo(TEST_LOCATION_NAME_ONE);
+
+        final Response responseGetAllLocations = doGetRequest(
+            BASE_LOCATIONS_URL,
+            headerMap
+        );
+        assertThat(responseGetAllLocations.getStatusCode()).isEqualTo(OK.value());
+        List<? extends Location> returnedLocations = Arrays.asList(responseGetAllLocations.getBody()
+                                                                       .as(Location[].class));
+
+        boolean testCourtOneReturned = returnedLocations.stream().anyMatch((location) -> TEST_LOCATION_NAME_ONE.equals(
+            location.getName()) && TEST_LOCATION_ID_ONE.equals(location.getLocationId())
+
+        );
+        boolean testCourtTwoReturned = returnedLocations.stream().anyMatch((location) -> TEST_LOCATION_NAME_TWO.equals(
+            location.getName()) && TEST_LOCATION_ID_TWO.equals(location.getLocationId())
+
+        );
+        boolean testCourtThreeReturned = returnedLocations.stream().anyMatch((location) ->
+                                                                                 TEST_LOCATION_NAME_THREE.equals(
+            location.getName()) && TEST_LOCATION_ID_THREE.equals(location.getLocationId())
+        );
+        assertTrue(testCourtOneReturned,"Should return " + TEST_LOCATION_NAME_ONE);
```

Comment:
You are using the wrong import for AssertJ:
import static org.assertj.core.api.AssertionsForClassTypes.assertThat;

Change it to the following import:
import static org.assertj.core.api.Assertions.assertThat;

After that you can simply assert like below by using assertThat + anyMatch directly:

 assertThat(returnedLocations).anyMatch(
            location -> TEST_LOCATION_NAME_ONE.equals(location.getName())
                && TEST_LOCATION_ID_ONE.equals(location.getLocationId())
        );